chainerx_cc/chainerx/routines/loss.cc

```diff
@@ -0,0 +1,59 @@
+#include "chainerx/routines/loss.h"
+
+#include <algorithm>
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include "chainerx/array.h"
+#include "chainerx/backend.h"
+#include "chainerx/backprop_mode.h"
+#include "chainerx/backward_builder.h"
+#include "chainerx/backward_context.h"
+#include "chainerx/constant.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/graph.h"
+#include "chainerx/kernels/math.h"
+#include "chainerx/macro.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/math.h"
+#include "chainerx/routines/type_util.h"
+#include "chainerx/scalar.h"
+#include "chainerx/shape.h"
+#include "chainerx/strides.h"
+
+namespace chainerx {
+
+Array MeanAbsoluteError(const Array& y, const Array& targ) {
```

Comment:
Are the variables names of the arguments following come convention? I'm wondering if we shouldn't stick to the ones similar to Chainer, i.e. `x0` and `x1`.

Reply:
Have renamed them as of now to be similar to `chainer`. However, from what I have seen in other frameworks, its generally `{y_pred, predictions, input}` and `{y_true, labels, target}` which I guess are better given that the function is used for calculating loss. Your thoughts?

Reply:
For routines that don't depend on the order such as MSE, let's stick with `x1`, `x2`. In cases the order matters, `x` and `t` is a convention quite often used by Chainer and we should stick to that one if possible. I see the point in having more verbose names too but it should be fine for common functions such as the ones in this PR.